app/src/main/java/net/skyscanner/backpack/demo/compose/BottomSheetStory.kt

```diff
@@ -75,6 +78,7 @@ fun BottomSheetStory(
 @ComposeStory(name = "Modal")
 fun ModalBottomSheetStory(
     modifier: Modifier = Modifier,
+    dragHandleStyle: BpkDragHandleStyle = BpkDragHandleStyle.OnImage(BpkDragHandleStyle.OnImage.Type.Dark),
```

Comment:
rather than modifying the existing one can you add a new variant which has the different styles? can you also modify the content in that case to show an image on top - both for the test to ensure the behaviour actually matches, but also to highlight how it should be used. there's various test images in the app module that you can use for this.
with a separate story you can also have an additional screenshot that needs to be added to the readme at the top

Reply:
Sure, i'll do that!